src/lib/api/core/base-adapter.ts

```diff
@@ -20,6 +20,16 @@ import {MediaMonitor} from '../../media-query/media-monitor';
 export class BaseFxDirectiveAdapter extends BaseFxDirective {
 
   /**
+   * Does this directive have 1 or more responsive keys defined
+   * Note: we exclude the 'baseKey' key (which is NOT considered responsive)
+   */
+  public hasResponsiveAPI() {
+    const totalKeys = Object.keys(this._inputMap).length;
+    const baseValue = this._inputMap[this._baseKey];
+    return  (totalKeys - (!!baseValue ? 1 : 0)) > 0;
```

Comment:
can this just be `return this.hasResponsiveAPI(this._baseKey)`?

Reply:
Sorry @mmalerba - I do not understand your question here ^.

Reply:
the `_baseKey` is the only non-responsive key. We want to check if 1 or more responsive keys are defined/in-use.

Reply:
I'm just saying that this class extends a class with a 1-parameter method of the same name, it looks like this no-parameter version could just call the 1-parameter version

Reply:
Fixed.